src/main/java/frc/robot/subsystems/Arm.java

```diff
@@ -80,6 +81,10 @@ public void setMotor(double armSpeed) {
         this.motorSpeedWidget.setDouble(armSpeed);
     }
 
+    public void setAngle(double angle) {
+        rotationController.setSetpoint(angle);
```

Comment:
to not compound error we need to reset the controller.
https://github.wpilib.org/allwpilib/docs/release/java/edu/wpi/first/math/controller/PIDController.html#reset()

do we need to reset the encoder as well?

Reply:
Let's not reset the encoder unless someone who knows more says we should/it causes problems not to.